dora/core/server/master/src/main/java/alluxio/master/job/DoraLoadJob.java

```diff
@@ -196,6 +195,7 @@ public DoraLoadJob(
    */
   public List<DoraLoadTask> prepareNextTasks() {
```

Comment:
make it private if not called by scheduler anymore

Reply:
done